dotnetapp-3.5/dotnetapp-3.5/Dockerfile-windowsservercore-1709

```diff
@@ -15,3 +15,11 @@ WORKDIR /app
 COPY . ./
 
 RUN ["msbuild.exe", "dotnetapp-3.5.csproj", "/p:Configuration=Release"]
+
```

Comment:
I find it difficult to spot the different phases of this dockerfile.  A couple things that may help is
1. Remove the WS between each instruction in the second phase.  The WS isn't adding value IMO. You may also collapse some of the WS in the builder phase.  Maybe group into two sections - install pre-reqs, build app.
2.  I've started putting two empty lines in-between phases as a way to visually separate them more.